python/py-rarfile/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-rarfile
+version             4.0
+revision            0
+
+categories-append   archivers
+platforms           darwin
+supported_archs     noarch
+license             ISC
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         RAR archive reader for Python
+long_description    ${description}
+
+homepage            https://github.com/markokr/rarfile
+
+checksums           rmd160  2be283cfabc9c6dc57735f773c858dbe2fd01e32 \
+                    sha256  67548769229c5bda0827c1663dce3f54644f9dbfba4ae86d4da2b2afd3e602a1 \
+                    size    148026
+
+python.versions     37 38 39
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
I think `setuptools` might need to be `depends_lib` for this module. Can you double-check?

Reply:
It works without setuptools:
```
 test-rarfile % wget https://unrarwindows.com/wp-content/uploads/2019/07/Wallpaper.rar
--2021-09-09 10:14:01--  https://unrarwindows.com/wp-content/uploads/2019/07/Wallpaper.rar
SSL_INIT
Resolving unrarwindows.com (unrarwindows.com)... 192.0.78.183, 192.0.78.213
Connecting to unrarwindows.com (unrarwindows.com)|192.0.78.183|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 49107263 (47M) [application/x-rar-compressed]
Saving to: ‘Wallpaper.rar’

Wallpaper.rar                        100%[===================================================================>]  46.83M  31.2MB/s    in 1.5s    

2021-09-09 10:14:03 (31.2 MB/s) - ‘Wallpaper.rar’ saved [49107263/49107263]

√ test-rarfile % port installed | grep py39-setuptools                                 
? test-rarfile % port installed | grep py39-rarfile                                    
  py39-rarfile @4.0_0 (active)
√ test-rarfile % python3.9                                                             
Python 3.9.7 (default, Sep  1 2021, 12:35:15) 
[Clang 12.0.5 (clang-1205.0.22.9)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import rarfile
>>> rf = rarfile.RarFile("Wallpaper.rar")
>>> for f in rf.infolist():
...     print(f.filename, f.file_size)
... 
beach-bird-s-eye-view-coast-50594.jpg 2503515
beach-bora-bora-clouds-753626.jpg 904586
beautiful-bloom-blooming-414160.jpg 4474545
beautiful-daylight-environment-709552.jpg 2230453
blur-branches-foliage-38537.jpg 3280945
color-depth-of-field-environment-1591447.jpg 3272836
dawn-hd-wallpaper-landscape-36717.jpg 192233
daylight-environment-forest-459225.jpg 4335581
4k-wallpaper-background-beautiful-853199.jpg 7936016
animal-animal-photography-blur-145939.jpg 1933652
animal-photography-animals-cold-39629.jpg 3853973
arch-bridge-clouds-814499.jpg 6322551
art-artistic-background-459301.jpg 6010598
asphalt-dark-dawn-531321.jpg 1931529
>>> 
```

@herbygillot why do you think so?

Reply:
it's not needed here indeed - perhaps @herbygillot was assuming that the port installed a `console_script`?